proknow/Users.py

```diff
@@ -314,60 +299,15 @@ def save(self):
                 jsmith.active = False
                 jsmith.save()
 
-            To update the user with a private role, set the `role_id` property to `None` and the
-            `role` property to a role definition::
-
-                from proknow import ProKnow
-
-                pk = ProKnow('https://example.proknow.com', credentials_file="./credentials.json")
-                jsmith = pk.users.find(email='jsmith@example.com')
-                jsmith.role_id = None
-                jsmith.role = {
-                    "create_api_keys": False,
-                    "manage_access": False,
-                    "manage_custom_metrics": False,
-                    "manage_template_metric_sets": False,
-                    "manage_renaming_rules": False,
-                    "manage_template_checklists": False,
-                    "organization_collaborator": False,
-                    "organization_read_patients": False,
-                    "organization_read_collections": False,
-                    "organization_view_phi": False,
-                    "organization_download_dicom": False,
-                    "organization_write_collections": False,
-                    "organization_write_patients": False,
-                    "organization_contour_patients": False,
-                    "organization_delete_collections": False,
-                    "organization_delete_patients": False,
-                    "workspaces": [],
-                }
-                jsmith.save()
-
         """
         body = {
             "email": self.email,
             "name": self.name,
             "active": self.active,
         }
-        assert (self.role_id is not None and self.role is None) or (self.role_id is None and self.role is not None), "exactly one of `role_id` or `role` is required"
-        if self.role_id is not None:
-            body["role_id"] = self.role_id
-        else: # self.role is not None:
-            body["role"] = self.role
-            for prop in ["id", "name", "private", "user", "created_at"]:
-                try:
-                    del body["role"][prop]
-                except KeyError:
-                    pass
         _, user = self._requestor.put('/users/' + self._id, json=body)
         self._data = user
         self.name = user["name"]
         self.email = user["email"]
         self.active = user["active"]
-        role = user["role"]
-        if role["private"] == True:
-            self.role_id = None
-            self.role = role
-        else:
-            self.role_id = role["id"]
-            self.role = None
+        
```

Comment:
Make sure to configure your IDE to use newlines at the end of files.

![Screen Shot 2022-02-17 at 9 42 41 AM](https://user-images.githubusercontent.com/6369365/154505063-9428f634-6c71-4e0f-b29b-6568c6d98aa4.png)